src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/ContactDetailsHelper.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.helper;
+
+import org.apache.commons.lang3.ObjectUtils;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.wrapper.ContactDetailsWrapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ContactDetailsHelper {
```

Comment:
Sorry I missed this before. I think it would be good to rename this to `ContactDetailsValidator` to keep its purpose specific.